R/utils_render_latex.R

```diff
@@ -365,9 +365,7 @@ create_footer_component_l <- function(data) {
     footnotes <-
       paste0(
         footnote_mark_to_latex(footnotes_tbl[["fs_id"]]),
-        footnotes_tbl[["footnotes"]] %>%
-          unescape_html() %>%
-          markdown_to_latex()
+        unlist(lapply(footnotes_tbl[["footnotes"]], process_text, context = "latex"))
```

Comment:
Actually can you change this to a `vapply` with `character(1)`? Since that shape is actually quite important.

Reply:
Good idea. Now fixed!